src/basics/Grid.js

```diff
@@ -111,10 +111,10 @@ const ColumnEl = styled.div`
   ${({ isIndependentScroll }) =>
     isIndependentScroll &&
     css`
-      overflow-y: scroll;
-      height: 100vh;
-      /* --vh will be set via setViewportHeight() ApiReference.js */
-      height: calc(var(--vh, 1vh) * 100);
+      @media (${MEDIA_QUERIES.gtTablet}) {
```

Comment:
This needs to be enabled in mobile for Api Reference to be able to scroll the side navigation

Reply:
The viewport setting change makes it so the sidenav takes up the entire left side of the page, I haven't needed to scroll. Page looks like this for me
<img width="417" alt="Screen Shot 2020-04-08 at 3 38 29 PM" src="https://user-images.githubusercontent.com/1551487/78826135-077b9a00-79af-11ea-8e81-6e1a19abac9f.png">

Reply:
Gotcha! One thing, all the sub items are supposed to be shown on mobile by default. But I just checked this site on my phone and its scroll and highlight functionality work pretty fine so let me confirm with Charles on whether we should display all subnav by default.

Reply:
Oh, I remember why I had to change this now. This style still applies on API Ref on mobile because the viewport is set to be wide, but having a scrollable div on mobile made /docs behave super weirdly. Was easy to get halfway past the nav, then start scrolling in the docs container while only half of it is onscreen.

Reply:
No need to display all nested items by default. All good 👍! Thank you.